data/satuan-karya-pramuka.js

```diff
@@ -0,0 +1,242 @@
+    pembangunan dalam rangka peningkatan pertahanan nasional.`,
+  sections: [
+    {
+      name: 'Organisasi',
+      description: `
+        Saka dapat dibentuk di "Kwartir Ranting" atas kehendak dan minat yang sama dari Pramuka Penegak dan
+        Pramuka Pandega, disesuaikan dengan situasu dan kondisi wilayahnya. Wewenang pengelolaan, pengendalian, dan 
+        pembinaan Saka berada di Kwartir Ranting, sedangkan pengesahan dilakukan oleh Kwartir Cabang. Apabila Kwartir
+        Ranting belum mampu membentuk Saka, maka pembentukan Saka dapat dilaksanakan oleh Kwartir Cabang yang wewenang, pengelolaan,
+        pengendalian, dan pembinaannya berada pada Kwartir Cabang. `,
+    },
+    {
+      name: 'Pembinaan',
+      description: `
+        Saka dibina oleh seorang Pamong Saka. Pamong Saka adalah Pembina Pramuka, terutama Pembina Pramuka Penegak/Pandega atau anggota dewasa lainnya, 
+        yang memiliki minat dalam satu bidang kegiatan Saka sesuai dengan minat anggota Saka yang bersangkutan. Pamong Saka diangkat dan dikukuhkan oleh 
+        Ketua Kwartir Cabang, atas usul Pimpinan Saka yang bersangkutan. Bila dalam Saka yang sejenis ada beberapa orang Pamong Saka, maka dipilih salah 
+        seorang sebagai kordinatornya. Masa bakti Pamong Saka adala tiga tahun dan sesudahnya dapat diangkat kembali. Pamong Saka secara ex-officio menjadi 
+        anggota Mabi Saka dari Saka yang bersangkutan.`,
+      groupType: 'object',
+      groups: [
+        {
+          text: 'Tugas dan Tanggung Jawab Pamong Saka',
+          lists: [
+            'Mengelola pembinaan dan pengembangan Sakanya',
+            'Menjadi Pembina Saka dan bekerja sama dengan Majelis Pembimbing Sakanya',
+            'Mengusahakan instruktur, perlengkapan dan keperluan kegiatan Sakanya',
+            'Mengadakan hubungan, konsultasi dan kerja sama yang baik dengan Pimpinan Saka, Kwartir, Majelis Pembimbing Saka, Gugusdepan dan Saka lainnya',
+            'Mengkoordinasikan instruktur dengan Dewan Kerja Saka yang ada dalam Sakanya',
+            'Menjadi anggota Mabi Saka',
+            'Menerapkan Prinsip Dasar dan Metode Kepramukaan serta sistem Among dalam kegiatan pembinaan Sakanya',
+            'Melaporkan perkembangan Sakanya kepada kwartir dan Mabi Saka yang bersangkutan',
+          ],
+        },
+      ],
+    },
+    {
+      name: 'Pelatihan',
+      description: `
+       Selain daripada Pamong Saka, untuk melatih anggota Saka dalam bidang Sakanya, maka di setiap Saka diadakan Instruktur Saka. Instruktur Saka adalah 
+       seseorang yang mempunyai kemampuan dan pengetahuan, keterampilan dan keahlian khusus di bidang tertentu yang bersedia membantu Pamong Saka dalam 
+       peningkatan kemampuan dan keterampilan anggotanya. Instruktur Saka diangkat dan dikukuhkan oleh Ketua Kwartir Cabang atas usul Pamong Saka dan Mabi Saka.`,
+      groupType: 'object',
+      groups: [
```

Comment:
Jangan pakai `object` sebagai key mas.
Ini reserved keyword di JS.
Better pakai kata lain.

Reply:
Changed to "group-object"